backend/api/communication/src/main/java/co/airy/core/api/communication/Stores.java

```diff
@@ -171,6 +184,10 @@ private void startStream() {
         return streams.acquireLocalStore(messagesStore);
     }
 
+    public ReadOnlyLuceneStore<String, Conversation> getConversationLuceneStore() {
+        return luceneProvider;
```

Comment:
There is no added benefit in accessing the Store using Kafka Streams, so we return the Lucene provider here so that we can query it directly.